tasks/mount.yml

```diff
@@ -0,0 +1,106 @@
+---
+# Perform a lookup if this module has not previously been invoked in that mode.
+- include: "lookup.yml"
+  when: os_manila_mount_host is not defined or
+        os_manila_mount_export is not defined or
+        os_manila_mount_access_key is not defined
+
+- name: Ensure supported protocol requested
+  fail:
+    msg: "Sorry, we only support CEPHFS right now."
+  when: os_manila_mount_share_protocol != "CEPHFS"
+
+- name: Install Ceph Repos
+  block:
+  - yum_repository:
+      name: ceph-noarch
+      description: Ceph - noarch
+      baseurl: "{{ os_manila_mount_ceph_repo_base }}/noarch"
+      gpgkey: "{{ os_manila_mount_ceph_repo_key | default(omit) }}"
+  - yum_repository:
+      name: ceph-arch
+      description: Ceph - arch
+      baseurl: "{{ os_manila_mount_ceph_repo_base }}/$basearch"
+      gpgkey: "{{ os_manila_mount_ceph_repo_key | default(omit) }}"
+  - yum: name=epel-release,ceph-common,ceph-fuse,fuse
```

Comment:
Don't you need to install epel-release separately from the other repos, otherwise you hit dependency issues on a fresh centos install? Now that is probably an existing bug.